### ISSUE TITLE
Cannot rename or delete files from S3 File Manager

### ISSUE THREAD

**Issue by tcarolan** [datagrok-ai/public#1861]
I have connected an S3 endpoint using the File Manger in Datagrok. I cannot delete or rename files. The permissions for the access key IAM user are correct. This has been validated using the S3 command line. Similarly, renaming/deleting files within the "Home" directory from the Datagrok specific S3 bucket also does not work. So, this is a general problem and not an S3 bucket specific problem. 

There is a corresponding Javascript Console error pasted below:

https://user-images.githubusercontent.com/6126004/235797066-27e38f8f-e41f-46c0-8790-60c2f360e15b.mp4

ml_dart2js.dart:3558 NullError: method not found: 'appendChild' on null
	https://datagrok.neomorph.com/packages/xamgle/src/views/files_view.dart 625:5                        <fn>
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 213:19     e1..a
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 237:5      e1..
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 187:19     dkV.
	https://datagrok.neomorph.com/packages/$sdk/lib/async/zone.dart 1128:37                              aYF
	https://datagrok.neomorph.com/packages/$sdk/lib/async/zone.dart 1012:12                              dat.Al
	https://datagrok.neomorph.com/packages/$sdk/lib/async/future_impl.dart 129:12                        ddP.
	https://datagrok.neomorph.com/packages/$sdk/lib/async/future_impl.dart 665:13                        Object.xO
	https://datagrok.neomorph.com/packages/$sdk/lib/async/future_impl.dart 343:9                         ddG.
	https://datagrok.neomorph.com/packages/$sdk/lib/async/zone.dart 1120:12                              aYD
	https://datagrok.neomorph.com/packages/$sdk/lib/async/zone.dart 1001:12                              dat.fD
	https://datagrok.neomorph.com/packages/$sdk/lib/async/zone.dart 901:14                               dat.Hv
	https://datagrok.neomorph.com/packages/$sdk/lib/async/zone.dart 926:20                               dau.
	https://datagrok.neomorph.com/packages/$sdk/lib/async/schedule_microtask.dart 41:5                   Object.dZ2
	https://datagrok.neomorph.com/packages/$sdk/lib/async/schedule_microtask.dart 50:5                   h3K
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 51:9       d9e.
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2456:48      fHX.
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/isolate_helper.dart 474:16  O9.LH
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/isolate_helper.dart 59:16   Object.G2
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2456:14      fHV
	https://datagrok.neomorph.com/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2476:14      MutationObserver.<fn>

**Comment by ahadzhanian** [comment]
Fixed in 1.16.0
![1861](https://github.com/datagrok-ai/public/assets/111368431/82b47363-5502-476d-9ff5-c63dbff0c838)

**Comment by ahadzhanian** [comment]
When trying to rename on public, an error occurs
![1861](https://github.com/datagrok-ai/public/assets/111368431/9d5218a4-868d-432c-8183-3207de19b037)

```
dart:755 NullError: method not found: 'appendChild' on null
	https://public.datagrok.ai/packages/xamgle/src/views/files_view.dart 614:5                        <fn>
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 213:19     e5q.a
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 237:5      e5q.
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 187:19     dpq.
	https://public.datagrok.ai/packages/$sdk/lib/async/zone.dart 1128:37                              b.x
	https://public.datagrok.ai/packages/$sdk/lib/async/zone.dart 1012:12                              deN.AF
	https://public.datagrok.ai/packages/$sdk/lib/async/future_impl.dart 129:12                        dis.
	https://public.datagrok.ai/packages/$sdk/lib/async/future_impl.dart 665:13                        Object.y0
	https://public.datagrok.ai/packages/$sdk/lib/async/future_impl.dart 343:9                         dij.
	https://public.datagrok.ai/packages/$sdk/lib/async/zone.dart 1120:12                              b.v
	https://public.datagrok.ai/packages/$sdk/lib/async/zone.dart 1001:12                              deN.fE
	https://public.datagrok.ai/packages/$sdk/lib/async/zone.dart 901:14                               deN.HX
	https://public.datagrok.ai/packages/$sdk/lib/async/zone.dart 926:20                               deO.
	https://public.datagrok.ai/packages/$sdk/lib/async/schedule_microtask.dart 41:5                   Object.e2s
	https://public.datagrok.ai/packages/$sdk/lib/async/schedule_microtask.dart 50:5                   haY
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/async_patch.dart 51:9       ddy.
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2456:48      fPa.
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/isolate_helper.dart 474:16  Os.Mr
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/isolate_helper.dart 59:16   Object.G6
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2456:14      fP8
	https://public.datagrok.ai/packages/$sdk/lib/_internal/js_runtime/lib/js_helper.dart 2476:14      MutationObserver.<fn>
```

**Comment by ahadzhanian** [comment]
Fixed in 1.16.2
